src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -0,0 +1,1790 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class Utf8StringsLiteralsTests : CompilingTestBase
+    {
+        private static string HelpersSource => @"
+class Helpers
+{
+    public static void Print(byte[] span)
+    {
+        System.Console.Write(""{"");
+        foreach (var item in span)
+        {
+            System.Console.Write("" 0x{0:X}"", item);
+        }
+        System.Console.WriteLine("" }"");
+    }
+
+    public static void Print(Span<byte> span)
+    {
+        System.Console.Write(""{"");
+        foreach (var item in span)
+        {
+            System.Console.Write("" 0x{0:X}"", item);
+        }
+        System.Console.WriteLine("" }"");
+    }
+
+    public static void Print(ReadOnlySpan<byte> span)
+    {
+        System.Console.Write(""{"");
+        foreach (var item in span)
+        {
+            System.Console.Write("" 0x{0:X}"", item);
+        }
+        System.Console.WriteLine("" }"");
+    }
+}
+";
+
+        [ConditionalFact(typeof(CoreClrOnly))]
```

Comment:
I didn't understand why this condition is specified on many of the tests. Is it not possible to execute code that uses the feature on desktop framework?

Reply:
> I didn't understand why this condition is specified on many of the tests. Is it not possible to execute code that uses the feature on desktop framework?

Documentation says "Span" types are not available on desktop.